### PR TITLE
k6 browser: Display Metrics and new Options page

### PR DESCRIPTION
**Pull request by ppcano**
1 -  New Browser options page that reference `script options` and environment variable options (migrated from `JavaScript API / k6 browser`).

Screenshot 2024-07-24 at 17 57 40

2 - Display the browser metric table on the Browser metrics page

Screenshot 2024-07-23 at 09 19 06

I'll port it to `next` if the k6 browser team approves the changes.

### CODE REVIEW

**Comment by ppcano** [comment]
@inancgumus, ready for your review, only available in `v0.52.x`

**Review by inancgumus — APPROVED** [review]
Great work 🙇 LGTM (besides the conflicting `_index.md`) 👍

**Comment by ppcano** [comment]
@heitortsergent, conflicts are fixed and ported to `next`. This PR is now ready to merge.

File: docs/sources/next/javascript-api/k6-browser/_index.md [hunk 1]
```diff
@@ -104,104 +104,7 @@ PS C:\k6> k6 run script.js
...
-| mute-audio      | `true`                                                                                      | Mutes audio sent to the audio device so it is not audible during automated testing.                                                                                                                                                                                                                                                                                                           |
-| blink-settings  | primaryHoverType=2,availableHoverTypes=2,<br />primaryPointerType=4,availablePointerTypes=4 | Sets blink settings. Format is <name\>[=<value\>],<name\>[=<value\>],... The names are declared in [settings.json5](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/frame/settings.json5) from chromium project. For boolean type, use "true", "false", or omit '=<value\>' part to set to true. For enum type, use the int value of the enum value. |
-
-## Devices Example
+### Devices Example
```
Comment (heitortsergent) [inline docs/sources/next/javascript-api/k6-browser/_index.md:107]:
```suggestion
### Devices example
```

File: docs/sources/next/javascript-api/k6-browser/_index.md [hunk 2]
```diff
@@ -240,3 +143,13 @@ export default async function () {
 ```
 
 {{< /code >}}
+
+## Browser-level APIs
+
+{{< docs/shared source="k6" lookup="javascript-api/k6-browser-api.md" version="<K6_VERSION>" >}}
+
+## Browser Module Options
```
Comment (heitortsergent) [inline docs/sources/next/javascript-api/k6-browser/_index.md:151]:
```suggestion
## Browser module options
```

File: docs/sources/next/using-k6-browser/options.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
+---
+title: 'Browser options'
+description: 'An overview of the different options to customize the behaviour when running browser tests.'
```
Comment (heitortsergent) [inline docs/sources/next/using-k6-browser/options.md:3]:
```suggestion
description: 'An overview of the different options you can use to customize the browser module behavior when running browser tests.'
```

File: docs/sources/next/using-k6-browser/options.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+| Option                    | Description                                                         |
+| ------------------------- | ------------------------------------------------------------------- |
+| type<sup>(required)</sup> | Name of the browser running the test. Options include `'chromium'`. |
+
+## Environment variable options
+
+You can customize the behavior of the browser module by passing environment variables.
+
+{{% admonition type="note" %}}
```
Comment (heitortsergent) [inline docs/sources/next/using-k6-browser/options.md:57]:
```suggestion
{{< admonition type="note" >}}
```

File: docs/sources/next/using-k6-browser/options.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+Customizing browser options via environment variables is unsupported when [running browser tests in Grafana Cloud k6](https://grafana.com/docs/grafana-cloud/testing/k6/get-started/run-your-first-browser-tests/).
+
+{{% /admonition %}}
```
Comment (heitortsergent) [inline docs/sources/next/using-k6-browser/options.md:61]:
```suggestion
{{< /admonition >}}
```

File: docs/sources/next/using-k6-browser/options.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+{{< docs/shared source="k6" lookup="browser/env-var-options.md" version="<K6_VERSION>" >}}
+
+### Default arguments
+
+List of default arguments included when launching the browser process. You can pass one or more of the arguments to the `K6_BROWSER_IGNORE_DEFAULT_ARGS` environment variable when starting a test for the ones you want to ignore.
+
+{{% admonition type="note" %}}
```
Comment (heitortsergent) [inline docs/sources/next/using-k6-browser/options.md:69]:
```suggestion
{{< admonition type="note" >}}
```

File: docs/sources/next/using-k6-browser/options.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+The starting '--' have been omitted from the argument names in these lists.
+
+{{% /admonition %}}
```
Comment (heitortsergent) [inline docs/sources/next/using-k6-browser/options.md:73]:
```suggestion
{{< /admonition >}}
```

**Review by heitortsergent — APPROVED** [review]

**Pull request merged**